src/main/java/de/muenchen/isi/domain/service/AbfrageService.java

```diff
@@ -94,12 +99,12 @@ public InfrastrukturabfrageModel saveInfrastrukturabfrage(final Infrastrukturabf
      *
      * @param abfrage zum Updaten
      * @return das geupdatete {@link InfrastrukturabfrageModel}
-     * @throws EntityNotFoundException falls die Abfrage identifiziert durch die {@link InfrastrukturabfrageModel#getId()} nicht gefunden wird
-     * @throws UniqueViolationException falls der Name der Abfrage {@link InfrastrukturabfrageModel#getAbfrage().getNameAbfrage} ()} bereits vorhanden ist
+     * @throws EntityNotFoundException    falls die Abfrage identifiziert durch die {@link InfrastrukturabfrageModel#getId()} nicht gefunden wird
```

Comment:
DataIntegrityViolationException ist nicht in den Javadoc der Methoden beschrieben.